roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -40,6 +40,13 @@ openshift_logging_es_pvc_prefix: "{{ openshift_hosted_logging_elasticsearch_pvc_
 # config the es plugin to write kibana index based on the index mode
 openshift_logging_elasticsearch_kibana_index_mode: 'unique'
 
+openshift_logging_elasticsearch_oauth_proxy_image_prefix: "openshift/oauth-proxy"
+openshift_logging_elasticsearch_oauth_proxy_image_version: "v1.0.0"
+openshift_logging_elasticsearch_oauth_proxy_cpu_limit: "100m"
+openshift_logging_elasticsearch_oauth_proxy_memory_limit: "32Mi"
+openshift_logging_elasticsearch_oauth_proxy_cookie_secret: "d4bc23207523aed802d9a3ef592fb4d9"
```

Comment:
? seems like a bad idea to hard code a secret in the config like this

Reply:
removed